app/src/main/java/ar/com/wolox/android/example/ui/auth/login/AuthFragment.kt

```diff
@@ -8,16 +9,41 @@ class AuthFragment : WolmoFragment<FragmentLoginBinding, AuthPresenter>(), AuthV
     override fun layout() = R.layout.fragment_login
 
     override fun init() {
+        val sharedPref = activity?.getPreferences(Context.MODE_PRIVATE) ?: return
+        val emailUser = sharedPref.getString("emailUser", "")
```

Comment:
You should define this string inside the companion object. For example:
const val EMAIL_USER = "email-user"